### ISSUE TITLE
confusion about frame_sample_mode

### ISSUE THREAD

**Issue by rockywind** [Brummi/BehindTheScenes#10]
Hi,
I am confusion about the setting of `frame_sample_mode`.
1. I don't know the meaning of the magical number of 4 and 2 [in the code block](https://github.com/Brummi/BehindTheScenes/blob/b866ad031e0a7c81c3a6067c99b6c604aae2c6fa/models/bts/trainer.py#L148).
2. I don't known the meaning of this for loop
````
                for cam in range(4):
                    ids_loss += [cam * steps + i for i in range(start_from, steps, 2)]
                    ids_render += [cam * steps + i for i in range(1 - start_from, steps, 2)]
                    start_from = 1 - start_from
````
3. When I train my dataset with only a front camera no stereo camera and fisheye camera, how can I modify these settings to fitting my dataset.
![image](https://user-images.githubusercontent.com/14838781/227956083-58033455-e631-42ae-a584-5b18de68de64.png)

**Comment by Brummi** [comment]
When training on Kitti-360, we have 4 different camera views at `steps` timesteps. -> 4 * `steps` frames.
The list of frames is ordered by camera.
`imgs = imgs_p_left + imgs_p_right + imgs_f_left + imgs_f_right` (from [kitti_360_dataset.py](https://github.com/Brummi/BehindTheScenes/blob/main/datasets/kitti_360/kitti_360_dataset.py))

To make training more stable, we partition the frames of each cameras separately. This is what happens in the four loop.
`frame_perm[0]` is a random number in [0, nv]. -> 50% chance to be even -> start_from is randomly chosen from {0, 1}.

To train on your own dataset, you can try the "default" frame sample mode, which randomly selects `n_frames_render` frames from the list of frames for color sampling. The rest of the frames are used to compute the loss.
Just make sure that the `n_frames_render` value is set adequately in the config. It should be around 50% of your total frame count.
